packages/transaction-controller/src/helpers/GasFeePoller.ts

```diff
@@ -320,7 +320,8 @@ export class GasFeePoller {
 }
 
 /**
- * Update the gas fees for a transaction.
+ * Updates gas properties for a transaction given `gasFeeEstimates` from the gas fee controller
```

Comment:
Not from the `GasFeeController` but from the gas fee flows, which may use the `GasFeeController` but we don't want to couple that here.